src/main/kotlin/no/nav/klage/oppgave/service/KlagebehandlingService.kt

```diff
@@ -132,6 +124,47 @@ class KlagebehandlingService(
         return klagebehandling
     }
 
+    fun createKlagebehandlingFromMottak(mottak: Mottak) {
+        if (klagebehandlingRepository.findByMottakId(mottak.id) != null) {
+            logger.error("We already have a klagebehandling for mottak ${mottak.id}, will not create a new one, nor update the existing one.")
+        }
+        val klagebehandling = klagebehandlingRepository.save(
+            Klagebehandling(
+                foedselsnummer = mottak.foedselsnummer,
+                tema = mottak.tema,
+                sakstype = mottak.sakstype,
+                referanseId = mottak.referanseId,
+                innsendt = null,
+                mottattFoersteinstans = null,
+                avsenderEnhetFoersteinstans = mottak.avsenderEnhet,
+                avsenderSaksbehandleridentFoersteinstans = mottak.avsenderSaksbehandlerident,
+                mottattKlageinstans = mottak.oversendtKaDato,
+                startet = null,
+                avsluttet = null,
+                frist = mottak.fristFraFoersteinstans,
+                tildeltSaksbehandlerident = mottak.tildeltSaksbehandlerident,
+                tildeltEnhet = mottak.tildeltEnhet,
+                mottakId = mottak.id,
+                vedtak = mutableSetOf(),
+                kvalitetsvurdering = null,
+                hjemler = mottak.hjemler().map { hjemmelService.generateHjemmelFromText(it) }.toMutableSet(),
+                saksdokumenter = if (mottak.journalpostId != null) {
+                    mutableSetOf(Saksdokument(journalpostId = mottak.journalpostId!!))
+                } else {
+                    mutableSetOf()
+                },
+                kilde = Kilde.OPPGAVE
```

Comment:
Kilde nå er sikkert litt uklar, men burde komme fra mottaket vel?

Reply:
Godt poeng. Kilde burde vi vel egentlig fjerne? Det er klage-oppgave-consumer (kabel-oppgave-broker?) som skal holde styr på dette vel, ikke kabal-api?

Reply:
Vet ikke om vi snakker om ulike ting her, men jeg tenker jo at `kilde` kommer med mottaket på en eller annen måte (kanskje utledes fra `tema`?). kabal-oppgave-broker skal vel kun håndtere oppgaver fra førsteinstanser som behandler i Gosys? D.v.s. ikke aktuell når vi får data fra en modernisert løsning.

Reply:
Kilde til nå har jo vært oppgave/ikke-oppgave, men du tenker at det bør gå over til å si noe om hvilke system klagen kommer fra? Ja, det kan jo være greit. Og da bør den jo komme fra mottaket isåfall, enig i det.